src/logger.test.js

```diff
@@ -0,0 +1,12 @@
+const logger = require("./logger");
+
+describe("test logger", () => {
+  it("give req, res and next", () => {
```

Comment:
Give names for tests according to standards, which I shared with all of you

Reply:
yes